style.css

```diff
@@ -87,3 +88,278 @@ body {
   justify-content: flex-start;
   padding-left: 5px;
 }
+
+.page1 {
+  background-color: var(--v3);
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 0%;
+  justify-content: center;
+  background-repeat: no-repeat;
+}
+
+.tituloMR {
+  font-style: normal;
+  font-size: 40px;
+  color: var(--v2);
+  font-family: 'Crete Round', serif;
+  padding-left: 15px;
+  margin-bottom: 0%;
+  padding-top: 802px;
+}
```

Comment:
- [ ] Kindly check your design with the Figma template you choose, as much as you can stick with it. The ` padding-top ` property value affects your design; Please fix it :wrench: `